composer.json

```diff
@@ -43,7 +43,7 @@
   "extra": {
     "downloads": {
       "phpstan": {
-        "url": "https://github.com/phpstan/phpstan/raw/master/phpstan.phar",
+        "url": "https://github.com/phpstan/phpstan/releases/latest/download/phpstan.phar",
         "path": "vendor/bin/phpstan",
```

Comment:
Can't we rather reference the latest version in composer and have dependabot/renovate update it when new releases of PhpStan come out?

Reply:
Unfortunately not, as we are not using Composer to pull in PHPStan. PHPStan requires PHP 7.1+, which would break our Composer build that requires PHP 5.6 support.

In case we ever bump the AMP plugin to PHP 7.1+, we can move over to doing that, though.